CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- Updated `.gitignore`
+- Updated `.gitignore` to track files with `*.bak` extension and to track `credentials.json` in any directory.
```

Comment:
Sorry for being nitpicky but I think this phrasing is unfortunate. You specified these files in `.gitignore` because you DO NOT want them to be tracked (by git), not because you want to track them :D.